__tests__/kata7.getEmployerRole.test.js

```diff
@@ -1,7 +1,28 @@
 const { getEmployerRole } = require('../src');
 
 describe('getEmployerRole', () => {
+const employees = [{
+
+    name: 'Holly',
+  
+    role: 'Category Manager'
+  
+  }, {
+  
```

Comment:
I don't think you need to leave empty spaces between lines here. This is just a style preference though.